docs/src/components/Sandpack.jsx

```diff
@@ -6,11 +6,89 @@ import {
   SandpackPreview,
 } from "@codesandbox/sandpack-react"
 
+import {
+  nextAuthCode,
+  indexCode,
+  layoutCode,
+  headerCode,
+  headerCss,
+  globalStyle,
+  sessionProviderCode,
+  // tsConfig,
+  // indexJsx,
+  // publicHtml,
+} from "./sandpackCode.js"
+
 export const CustomSandpack = () => (
-  <SandpackProvider template="react">
-    <SandpackLayout>
-      <SandpackCodeEditor />
-      <SandpackPreview />
+  <SandpackProvider
+    customSetup={{
+      dependencies: {
+        react: "17.0.2",
+        "react-dom": "17.0.2",
+        next: "^12.0.11-canary.4",
```

Comment:
Hey, I'm sorry if I didn't make myself very clear in [this commentary](https://github.com/codesandbox/sandpack/issues/359#issuecomment-1031513557) and [here](https://sandpack.codesandbox.io/docs/faq#does-sandpack-support-nextjs-or-other-server-side-applications). Sandpack doesn't support any dependency that needs a server to run (but only runtime projects). I thought you had different purposes for Sandpack here, so I'm afraid to say that CodeSandbox Embed is still the best option.

I also addressed this issue on Sandpack project, to avoid this kind of assumption again https://github.com/codesandbox/sandpack/pull/410

Reply:
Ah yes you're right, must have forgotten that piece. Thanks anyway for all the support so far!